config/nova-careers.php

```diff
@@ -42,4 +42,14 @@
     |
     */
     'excluded_blocks' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Traffic Cop
+    |--------------------------------------------------------------------------
+    |
+    | Indicates whether Nova should check for modifications between viewing and updating a resource.
```

Comment:
@liam-spedding Can we keep the formatting correct according to how laravel does this for config files?

The description of the config variable isn't usually longer than the dashes at the top but can flow onto multiple lines.